CHANGELOG

```diff
@@ -12,7 +12,8 @@ Added
 
 Fixed
 
-- None
+- SQL error in Rails Conditional GET (304 caching)
+  [#295](https://github.com/collectiveidea/audited/pull/295)
```

Comment:
What do you think about converting the file to markdown? It displays nicely in github, with clickable links. We could just add a `.md` extension.